projects/observability/src/shared/components/cartesian/chart.ts

```diff
@@ -66,6 +66,7 @@ export interface Band<TInterval> {
 export interface GroupByFilter {
```

Comment:
What's a "groupByFillter"? Was this added recently? Given there's no operator or value here, guessing this is just misnamed and describes a group by, not a filter.

Reply:
renamed to groupBy